packages/@aws-cdk/aws-dynamodb/test/integ.global-replicas-provisioned.ts

```diff
@@ -1,3 +1,5 @@
+/// !cdk-integ pragma:ignore-assets
```

Comment:
Why?

Reply:
The general thought is that we don't want asset hash changes to force an update of the tests.

Reply:
I think ignoring assets is not a healthy approach. Asset hashes usually represent runtime code that is an intrinsic part of custom resources. We actually want to require that integ tests will get re executed if the runtime code changes.

Reply:
I agree, this seems dangerous to just apply globally. We can `dry-run` the integ test individually if we think all changes are just harmless asset changes.

Reply:
> I think ignoring assets is not a healthy approach. Asset hashes usually represent runtime code that is an intrinsic part of custom resources. We actually want to require that integ tests will get re executed if the runtime code changes.

Then why did we add this `/// !cdk-integ pragma:ignore-assets` capability to our integ tests...?

Reply:
> I agree, this seems dangerous to just apply globally.

But it's not globally, it's set per integ test...?

Reply:
> But it's not globally, it's set per integ test...?

Poor choice of words. Maybe "blindly"? :)

Reply:
> Then why did we add this

Good question. Worth revisiting that decision I think

Reply:
> Then why did we add this /// !cdk-integ pragma:ignore-assets capability to our integ tests...?

Sometimes assets are used in the test itself and then they could be inconsequential

Reply:
Changed.